components/kyma-environment-broker/common/orchestration/dto.go

```diff
@@ -8,13 +8,18 @@ import (
 
 // Parameters hold the attributes of orchestration create (upgrade) requests.
 type Parameters struct {
-	Targets  TargetSpec   `json:"targets"`
-	Strategy StrategySpec `json:"strategy,omitempty"`
-	DryRun   bool         `json:"dryRun,omitempty"`
+	Targets    TargetSpec           `json:"targets"`
+	Strategy   StrategySpec         `json:"strategy,omitempty"`
+	DryRun     bool                 `json:"dryRun,omitempty"`
+	Kubernetes KubernetesParameters `json:""`
 	// upgrade kyma specific parameters
 	Kyma KymaParameters `json:""`
 }
 
+type KubernetesParameters struct {
+	Version string `json:"kubernetesVersion,omitempty"`
+}
+
 // KymaParameters hold the attributes of kyma upgrade specific orchestration create requests.
 type KymaParameters struct {
 	Version string `json:"kymaVersion,omitempty"`
```

Comment:
Since we now don't embed Kyma into Parameters, we can shorten the JSON field name in Kyma:

```suggestion
	Version string `json:"version"`
```